src/store/channels-list/api.ts

```diff
@@ -1,13 +1,10 @@
-export const api = {
-  async fetch(_id: string) {
-    return [
-      'stuff',
-      'tacos',
-      'cats',
-      'work',
-    ].map((name, index) => ({
-      id: `channel-id-${index}`,
-      name,
-    }));
-  },
-};
+import { config } from '../../config';
+
+export async function fetchChannels(_id: string) {
+  const data = await fetch(`${config.ZERO_API_URL}/api/networks/${_id}/chatChannels/public`);
+  const channels = await data.json();
+  return channels.map((channel, index) => ({
+    id: `channel-id-${index}`,
+    ...channel,
+  }));
```

Comment:
let's pull this into the saga so that we can easily test it. additionally, we should map the actual channel id, rather than generating one. double-check this, but i believe the id field on channels is `url`.